lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-cloud_manager-vm.rb

```diff
@@ -2,6 +2,14 @@ module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_CloudManager_Vm < MiqAeServiceVmCloud
     expose :cloud_networks, :association => true
 
+    expose :resize
+    expose :confirm_resize
```

Comment:
It does not, more precisely the returned object is not useful for following processing.